### DISCUSSION TITLE
Alpha 20230129_0235P

### DISCUSSION THREAD

**Discussion by EmiyaSyahriel** [EmiyaSyahriel/CrossLauncher#28] [General]
## What's New
- Package Installer, easily installs your customization package (in form of XPKG file).
- Custom Resolution, have to support those weird aspect ratios.
- System Bar Visibility, the launcher already have a status bar!
- Overflow scrolling texts, when there is not enough space for the text

## Fixed
- Memory leak when drawing arrow, changes vector path rendering to bitmap-based rendering, IDK why.
- No touch interaction in app info dialog
- Some settings set wrong value to preferences
- System orientation now uses right description
- Category arrangement dialog now shows rectangle to signify which item is now active

## Removed
- Reference Resolution Menu, replaced by Reference Resolution dialog instead.

Sample XPKG File : [ClashOfClans.xpkg](https://drive.google.com/file/d/1Awbwh-ZN9GvTk8KouwZFFRNobWoREYvx/view?usp=share_link)

[**Preview Video** ![Thumbnail](https://img.youtube.com/vi/rHcZDoyGuWg/default.jpg) ](https://youtu.be/rHcZDoyGuWg)

**Full Changelog**: https://github.com/EmiyaSyahriel/CrossLauncher/compare/a_20221218...a_20230129

<hr /><em>This discussion was created from the release <a href='https://github.com/EmiyaSyahriel/CrossLauncher/releases/tag/a_20230129'>Alpha 20230129_0235P</a>.</em>

**Comment by GoodDevil666** [comment]
Bro I installed on my phone and it does not start for me. Smartphone model Xiaomi Redmi Note 4X Android 7.0

**Reply by EmiyaSyahriel** [reply]
May I know on how it does not start on your device? Is it closing right away or it opens for several seconds and then closing.

Or do you have any logcat (android log info) regarding to this problem, so I can see a better view on what kind of error the launcher stumble upon when it's trying to run on your device?
